mkdocs.yml

```diff
@@ -71,11 +71,11 @@ pages:
   # API Permissions: api/permissions.md         # page-tree = NEED_NEW_PAGE
   - API Chaining: api/chaining.md
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
+- Setup:
```

Comment:
This is almost what I had in mind. Sorry to be nit-picky, but can you put "Setup" within "Hooks"? Is that okay? Does that make sense?